code/iaas/service-discovery/server/src/main/java/io/cattle/platform/servicediscovery/process/ServiceDeactivate.java

```diff
@@ -22,15 +16,6 @@
 public class ServiceDeactivate extends AbstractObjectProcessHandler {
 
     @Inject
-    ObjectManager objectManager;
-    
-    @Inject
-    ResourceMonitor resourceMonitor;
-
-    @Inject
-    ProcessProgress progress;
-
-    @Inject
     ServiceDiscoveryService sdServer;
```

Comment:
Is `sdServer` the right name for this? Did you mean `sdService`?
...Won't stop me from LGTMing, just wanted to point it out.